modules/social_features/social_core/src/ContentTranslationDefaultsConfigOverride.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace Drupal\social_core;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Extension\ModuleHandler;
+
+/**
+ * Provides content translation for the Social Core module.
+ *
+ * @package Drupal\social_core
+ */
+class ContentTranslationDefaultsConfigOverride implements ConfigFactoryOverrideInterface {
+
+  /**
+   * The module handler.
+   *
+   * @var \Drupal\Core\Extension\ModuleHandlerInterface
+   */
+  protected $moduleHandler;
+
+  /**
+   * Constructs the service with DI.
+   *
+   * @param \Drupal\Core\Extension\ModuleHandler $module_handler
+   *   The module handler.
+   */
+  public function __construct(ModuleHandler $module_handler) {
+    $this->moduleHandler = $module_handler;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function loadOverrides($names) {
+    $overrides = [];
+
+    // If the module "content_translation" is enabled let make translations
+    // enabled for content provided by the module by default.
+    $is_content_translations_enabled = $this->moduleHandler
+      ->moduleExists('content_translation');
```

Comment:
As in other extensions we need to enable translations here if only `social_content_translation` is enabled.